clang/test/Sema/conversion.c

```diff
@@ -443,4 +443,4 @@ float double2float_test4(double a, float b) {
   b -= a; // expected-warning {{implicit conversion when assigning computation result loses floating-point precision: 'double' to 'float'}}
   return b;
 }
-float f = 1.0 / 2.0; // expected-warning {{implicit conversion between floating point types of different sizes}}
```

Comment:
From a quick look, it does not look like this test is present in community clang's version of this file. If that is the case, I would just remove this.